ocaml/sdk-gen/component-test/spec/go_test.json

```diff
@@ -0,0 +1,877 @@
+{
+  "test_id1": {
+    "method": [
+      "session.login_with_password",
+      "pool.get_all",
+      "pool.get_record",
+      "host.get_record",
+      "session.logout"
+    ],
+    "params": {
+      "session.login_with_password": ["", "", "1.0", "Go SDK component test"],
+      "pool.get_all":  ["OpaqueRef:12345678-4d8a-e4a9-a241-1ea05c1d6707"],
+      "pool.get_record": ["OpaqueRef:12345678-4d8a-e4a9-a241-1ea05c1d6707", "OpaqueRef:9c81868a-4d8a-e4a9-a241-1ea05c1d6707"],
```

Comment:
Why is there one record for `poo.get_all`, but two records for `pool.get_record`?